src/app-layout/drawer/index.tsx

```diff
@@ -204,51 +198,51 @@ export const DrawerTriggersBar = ({
         className={clsx(styles['drawer-content'])}
       >
         {!isMobile && (
-          <aside aria-label={drawers?.ariaLabel} className={clsx(styles['drawer-triggers-wrapper'], contentClassName)}>
+          <aside
+            aria-label={drawers?.ariaLabel}
+            className={clsx(styles['drawer-triggers-wrapper'], testutilStyles['drawers-desktop-triggers-container'])}
+          >
             <>
-              {visibleItems?.map((item: DrawerItem, index: number) => {
+              {visibleItems.map((item, index) => {
                 return (
-                  <span
+                  <DrawerTrigger
```

Comment:
Extracted utility component to reuse between normal triggers and the overflow trigger